modules/administration/pages/tshoot-taskomatic.adoc

```diff
@@ -19,7 +19,13 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 ////
 
 If Taskomatic crashes, repository metadata regeneration can be prevented from happening regularly.
```

Comment:
```suggestion
If Taskomatic crashes, repository metadata regeneration can be interrupted.
```
The other wording sounds strange to my foreign ears.  At first, I had doubt whether that's a good thing or not.  But this is just a proposal, feel free to ignore it!